activation/atxdb.go

```diff
@@ -94,6 +96,50 @@ func (m *ActivationDb) CalcActiveSetFromView(a *types.ActivationTx) (uint32, err
 
 }
 
+const NIPST_LAYERTIME = 1000
+
+func (db *ActivationDb) ValidateAtx(atx *types.ActivationTx) error {
```

Comment:
There are inconsistent receiver names in this module. These methods are different than the rest:
```golang
func (m *ActivationDb) ProcessBlockATXs(blk *types.Block) {
```
```golang
func (m *ActivationDb) CalcActiveSetFromView(a *types.ActivationTx) (uint32, error) {
```